src/AstariaRouter.sol

```diff
@@ -301,6 +301,9 @@ contract AstariaRouter is
 
     if (what == FileType.AuctionWindow) {
       uint256 window = abi.decode(data, (uint256));
+      if (window == 0) {
+        revert InvalidFileData();
```

Comment:
we need the same check as below to ensure that we cant set the window above the min epoch length